File: OfficeRadio/Core.Models/Models/Audio.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.Models
{
    public class Audio
    {
        public int Id { get; set; }
        public string Artist { get; set; }
        public string Album { get; set; }
        public string Duration { get; set; }
        public string Genre { get; set; }
        public string ImageUrl { get; set; }
        public string Name { get; set; }
        public int NoOfRequests { get; set; }
        public int[] Emotions { get; set; }
        public string Path { get; set; }

    }
}

File: OfficeRadio/OfficeRadios/OfficeRadio/src/app/models/Audio.ts

export class Audio {
  id: number;
  artist: string;
  album: string;
  duration: string;
  genre: string;
  imageUrl: string;
  name: string;
  noOfRequests: number;
  emotions: number[];
  path: string;
  constructor(args: Audio) {
    this.id = args.id;
    this.artist = args.artist;
    this.album = args.album;
    this.duration = args.duration;
    this.genre = args.genre;
    this.imageUrl = args.imageUrl;
    this.name = args.name;
    this.noOfRequests = args.noOfRequests;
    this.emotions = args.emotions;
    this.path = args.path;
  }
}
